### PR TITLE
fix: repair fdr data for support analysis 

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes
Unfortunately #6913 removed many vital data points in the fdr files without adequate replacement. This renders FDR files useles for support purposes (AP Disconnect Analysis, COntroller Input Analysis, High AOA Analysis, etc.). 

This PR attempts to bring these data points back into the fdr files. 

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
Testing the FDR with this PR only is not possible. It also requires the coversion tool and graph tool. 

But it shoul be test against general issues - basically anything should work as before. Probably best to simply be used in Exp. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/fbw/src/FlyByWireInterface.h [hunk 1]
```diff
@@ -94,6 +97,26 @@ class FlyByWireInterface {
...
+  double ambient_wind_velocity_kn = 0.0;
+  double ambient_wind_direction_deg = 0.0;
+  double total_air_temperature_celsius = 0.0;
+  // aoa
+  double alpha_floor_condition = 0.0;
+  // these are not correct yet
+  double protection_ap_disc = 0.0;
+  double v_alpha_prot_kn = 0.0;
+  double v_alpha_max_kn = 0.0;
+
```
Comment (lukecologne) [inline src/fbw/src/FlyByWireInterface.h:119]: Seems like these fields are not used and are duplicated from [AdditionalData.h](https://github.com/flybywiresim/a32nx/pull/7538/files#diff-6a74ba4307cc2097b8e85392beee956d961c74424e0d2192497afbc6c09bf88f)

File: src/fbw/src/failures/FailuresConsumer.cpp [hunk 2]
```diff
@@ -50,3 +50,12 @@ bool FailuresConsumer::setIfFound(double identifier, bool value) {
     return true;
   }
 }
+
+bool FailuresConsumer::isAnyActive() {
+  if (std::any_of(activeFailures.begin(), activeFailures.end(), [](auto pair) {
```
Comment (lukecologne) [inline src/fbw/src/failures/FailuresConsumer.cpp:55]: Vscode tells me that `std::any_of` needs to have `algorithm` included? Not sure why it compiles without it

**Review by lukecologne — CHANGES_REQUESTED** [review]

File: src/fbw/src/failures/FailuresConsumer.cpp (hunk 2 above)
Comment (frankkopp) [reply src/fbw/src/failures/FailuresConsumer.cpp:55]: My IDE shows that it seems to be included down the line somewhere - but maybe better to add it explicitly.

File: src/fbw/src/FlyByWireInterface.h (hunk 1 above)
Comment (frankkopp) [reply src/fbw/src/FlyByWireInterface.h:119]: yea - my fault - interestingly these inits seem to not even be necessary. 
Will fix

**Review by lukecologne — APPROVED** [review]
Looks good!

**Pull request merged**